proftpd/sftp.sls

```diff
@@ -0,0 +1,26 @@
+{% from "proftpd/map.jinja" import proftpd with context %}
+
+proftpd_sftp_config:
```

Comment:
the two following states suffer from the same 'non-immutable' status than indicated below for modules.conf

Reply:
I add test before like file.contains and file.access